SoftwareCo/SoftwareCo/SoftwareOpenCodeMetricsTreeCommand.cs

```diff
@@ -87,7 +87,6 @@ private static void Execute(object sender, EventArgs e)
             try
             {
                 CodeMetricsTreeManager.Instance.OpenCodeMetricsPaneAsync();
-                EventManager.Instance.CreateCodeTimeEvent("mouse", "click", "ShowTreeView");
```

Comment:
we should still track this event.   but with the correct event names and locations.     @bjacobson26 added one for this in Sublime